client-sdk/go/modules/rofl/policy.go

```diff
@@ -39,8 +43,46 @@ type AllowedEndorsement struct {
 type FeePolicy uint8
 
 const (
-	// FeePolicyAppPays is a fee policy where the application enclave pays the gas fees.
-	FeePolicyAppPays FeePolicy = 1
+	// FeePolicyInstancePays is a fee policy where the application enclave pays the gas fees.
+	FeePolicyInstancePays FeePolicy = 1
 	// FeePolicyEndorsingNodePays is a fee policy where the endorsing node pays the gas fees.
 	FeePolicyEndorsingNodePays FeePolicy = 2
 )
+
+// UnmarshalYAML implements yaml.Unmarshaler.
+func (fp *FeePolicy) UnmarshalYAML(value *yaml.Node) error {
+	switch value.ShortTag() {
+	case "!!str":
```

Comment:
Why do we need to support this case?  Since encoding is not unique, one might mix different encodings :thinking: 

```yaml
fee_policy1: instance
fee_policy2: 1
```

Reply:
Just for compatibility. Maybe no need to support it.

Reply:
Fixed.